zubhub_frontend/zubhub/src/views/edit_profile/editProfileScripts.js

```diff
@@ -219,18 +207,17 @@ export const handleTooltipClose = () => {
  * @todo - describe object's function
  */
 export const validationSchema = Yup.object().shape({
-  username: Yup.string().required('required'),
-  user_location: Yup.string().min(1, 'min').required('required'),
-  password: Yup.string().required('required'),
-  email: Yup.string().email('invalid').when('phone', {
-    is: (phone) => !phone || phone.length === 0,
-    then: Yup.string().required('phoneOrEmail')
-  }),
-  phone: Yup.string().when('email', {
-    is: (email) => !email || email.length === 0,
-    then: Yup.string().required('phoneOrEmail')
-  }).test('phone_is_invalid', 'invalid', function (value) {
-    return /^[+][0-9]{9,15}$/g.test(value) || !value ? true : false;
-  }),
-  bio: Yup.string().max(255, 'tooLong'),
-}, ['phone', 'email']);
\ No newline at end of file
+  username: Yup.string().required('Username is required'),
+  user_location: Yup.string().required('Location is required'),
+  password: Yup.string().required('Password is required'),
+  email: Yup.string().email('Invalid email'),
```

Comment:
`.when('phone', ` was removed from here. We should put it back

Reply:
also see this https://github.com/unstructuredstudio/zubhub/pull/1089#discussion_r1477551368